postgresql11.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: postgresql
+  name: postgresql11
```

Comment:
`postgresql11` or `postgresql-11 `?

Reply:
Personally I like the "-" but I wonder if there's a reason Alpine don't have it...

Reply:
debian (apt) seems to use a -

Reply:
We can use `-` or no dash.  `@` is restricted for tagged repositories.

Reply:
Do one of you two want to make a call as I don't have an opinion?  Then I'll leave or change this PR.  

I've got a followup for Java ready so wont push that until I know if we are using a dash or not.